tools/ci_build/build.py

```diff
@@ -1516,11 +1516,17 @@ def run_onnxruntime_tests(args, source_dir, ctest_path, build_dir, configs):
                     run_subprocess([sys.executable, '-m', 'unittest', 'discover', '-s', 'quantization'],
                                    cwd=cwd, dll_path=dll_path)
                     if args.enable_transformers_tool_test:
+                        import numpy
+                        numpy_init_version = numpy.__version__
                         required = {
-                            'numpy==1.19.2', 'coloredlogs==15.0', 'tf2onnx==1.8.5', 'transformers==4.6.1',
-                            'torch==1.8.1', 'tensorflow==2.5.0', 'onnxconverter-common==1.8.1', 'psutil'}
+                            'numpy==1.19.2', 'coloredlogs==15.0', 'transformers==4.6.1', 'torch==1.8.1',
+                            'onnxconverter-common==1.8.1', 'psutil'}
                         run_subprocess([sys.executable, '-m', 'pip', 'install', *required])
                         run_subprocess([sys.executable, '-m', 'pytest', 'transformers'], cwd=cwd)
+                        # Restore initial environment
+                        run_subprocess([sys.executable, '-m', 'pip', 'uninstall', *required, '-y'])
```

Comment:
Ideally, I think it might be better to setup and restore(uninstall) environment inside a specific test. When we update the e.g torch version we can make sure it won't break other tests(If they have a required torch version)